charts/jenkins/values.yaml

```diff
@@ -225,7 +226,13 @@ controller:
     # Ignored if securityRealm is defined in controller.JCasC.configScripts and
     # ignored if controller.enableXmlConfig=true as controller.securityRealm takes precedence
     securityRealm: |-
-      legacy
+      local:
+        allowsSignup: false
+        enableCaptcha: false
+        users:
+        - id: "${chart-admin-username}"
```

Comment:
won't this expand to empty by default? is that what we want?

Reply:
> The secret needs to be mounted as a file to /run/secrets/, and then the filename can be used as the KEY.https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/features/secrets.adoc#kubernetes-secrets

I added a comment where the secret is mounted. As long as `controller.adminSecret` everything should be fine.